docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -92,49 +277,78 @@ See [provider contracts](../contracts/overview.md) for more details.
   - `MachinePoolPhase`, used in `machinePool.status.phase`
   - `MachinePhase`, used in `machine.status.phase`
   
-### Removals
+## Removals
 
 There are no removals in the CAPI v1.11 version.
 
-As documented in [Suggested changes for providers](#suggested-changes-for-providers), please also start planning for future removals:
+However, as documented in [Suggested changes for providers](#suggested-changes-for-providers), it is highly reccomanded to start planning for future removals:
 
 - v1beta1 API version will be removed tentatively in August 2026
 - Starting from the CAPI release when v1beta1 removal will happen (tentative Aug 2026), the Cluster API project
-  will remove the Cluster API condition type, the `util/conditions` package, the code handling conditions in `util/patch.Helper`
-  and everything related to the custom Cluster API `v1beta.Condition` type.
+  will remove the Cluster API condition type, the `util/conditions/deprecated/v1beta1` package, 
+  the `util/deprecated/v1beta1` package package, the code handling conditions in `util/patch.Helper`
```

Comment:
2x package

> the code handling conditions in `util/patch.Helper`

only the old conditions, right?

Reply:
yes, fixed everywhere